File: backend/notification-service/src/main/java/com/familring/notificationservice/model/dto/response/NotificationResponse.java
package com.familring.notificationservice.model.dto.response;

import com.familring.notificationservice.model.dto.NotificationType;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class NotificationResponse {
    private Long notificationId;
    private Long receiverUserId;
    private Long senderUserId;
    private String destinationId;
    private NotificationType notificationType;
    private String notificationTitle;
    private String notificationMessage;
    private boolean notificationIsRead;
}

File: android/domain/src/main/java/com/familring/domain/model/notification/NotificationResponse.kt
package com.familring.domain.model.notification

data class NotificationResponse(
    val notificationId: Long,
    val receiverUserId: Long,
    val senderUserId: Long,
    val destinationId: String,
    val notificationType: String,
    val notificationTitle: String,
    val notificationMessage: String,
    val notificationIsRead: Boolean,
)
